Dockerfile

```diff
@@ -89,11 +82,10 @@ RUN docker-php-ext-install -j$(nproc) bz2 &&\
   docker-php-ext-install imap &&\
   docker-php-ext-install mysqli pdo pdo_mysql &&\
   docker-php-ext-install zip &&\
-  docker-php-ext-configure pcntl --enable-pcntl && \
-  docker-php-ext-install pcntl &&\
-  docker-php-ext-enable oci8 &&\
```

Comment:
Same here, why are you removing these?

Reply:
We use a (fake) MySQL database nowadays instead of Oracle during a build.